task-1/task1.2.js

```diff
@@ -0,0 +1,65 @@
+import csv from 'csvtojson';
+import fs from 'fs';
+import { pipeline, Transform } from 'stream';
+import { createInterface } from 'readline';
+
+
+const csvFilePath= './task-1/csv/nodejs-hw1-ex1.csv';
+const outputFilePath = './task-1/csv/csv-to-txt.txt';
+
+const stringTransform = new Transform();
+stringTransform._transform = function(chunk, encoding, done) {
+    const stringToObj = JSON.parse(chunk.toString());
+    const newObj = {};
+    Object.entries(stringToObj).forEach(([key, value]) => {
+        if (key === 'Amount') {
+            return;
+        }
+        if (key === 'Price') {
+            newObj[key.toLowerCase()] = Number(value);
+            return;
+        }
+        newObj[key.toLowerCase()] = value;
+    });
+    done(null, JSON.stringify(newObj) + '\n');
```

Comment:
JSON `parse` and `stringify` are expensive. So, I recommend you to avoid using them if there is other way to implement task, especially if good performance is important